Migrations/create.recipe.json

```diff
@@ -98,6 +98,24 @@
                   "Position": "0"
                 }
               }
+            },
+            {
+              "PartName": "CommonPart",
+              "Name": "CommonPart",
+              "Settings": {
+                "ContentTypePartSettings": {
+                  "Position": "3"
+                },
+                "CommonPartSettings": {
+                  "DisplayDateEditor": true
+                },
+                "LinkableContentTypes": [],
```

Comment:
Don't think `LinkableContentTypes` & `GalleryPartSettings` should be here. Assume this addition is from an export so it's a bit weird that these are included.